assets/app.js

```diff
@@ -621,11 +559,21 @@ new Vue({
             }
         },
         translationsComponents: function () {
+            if(this.rawCompoent == null){
+                this.rawCompoent = ComponentOptions.content
+            }
+
             if (ComponentOptions && ComponentOptions.content) {
                 var response = []
                 ComponentOptions.content.forEach(function (option, index) {
-                    if(option.name !== '-' && option.is_active==1){
-                        response.push(option);
+                    if(IsAdmin == true){
```

Comment:
if you extract this bit into a separated function, you could just use it and your code could `DRY` a little bit :slightly_smiling_face:

Reply:
Same as on `translationsBaseProducts`: put in a dedicated function `translatorComponents`